fastlane/lib/fastlane/runner.rb

```diff
@@ -119,14 +130,12 @@ def trigger_action_by_name(method_sym, custom_dir, from_action, *arguments)
       # First, check if there is a predefined method in the actions folder
       class_ref = class_reference_from_action_name(method_sym)
       unless class_ref
-        alias_found = find_alias(method_sym.to_s)
-        if alias_found
+        class_ref = class_reference_from_action_alias(method_sym)
+        # notify action that it has been used by alias
+        if class_ref.respond_to?(:alias_used)
           orig_action = method_sym.to_s
-          class_ref = class_reference_from_action_name(alias_found.to_sym)
-          # notify action that it has been used by alias
-          if class_ref.respond_to?(:alias_used)
-            class_ref.alias_used(orig_action, arguments.first)
-          end
+          arguments = [{}] if arguments.empty?
```

Comment:
@hjanuschka Thanks for reviewing.

It initializes an array of its first element as hash. 👍 

```ruby
irb(main):001:0> Array.new(1, Hash.new)
=> [{}]
```